server/persistence/mysql/consts.go

```diff
@@ -7,11 +7,11 @@ import (
 )
 
 const (
-	maxPlayerUUIDLen    = 255
-	maxPlayerUUIDLenStr = `255`
+	maxPlayerUUIDLen    = 191
```

Comment:
I saw that the max number allowed in an index is 191 with utfmb4 (rather than 255 for utf8) in [this article](https://mathiasbynens.be/notes/mysql-utf8mb4): `for utf8 or utf8mb4 columns, you can index a maximum of 255 or 191 characters`.